wpeverest/user-registration
# Enhance - Redirection after registration

### All Submissions:






### Changes proposed in this Pull Request:

This PR enhances the existing feature of redirection after registration. 

Previously, we could only add a redirect URL to the form setting. Now, there is a completely new setting where the admin can choose various redirect options like:
1. No redirection.
2. Redirect to an internal page.
3. Redirect to the external URL.

Closes # .

### How to test the changes in this Pull Request:
**_Note: Please check for backward compatibility before testing new features._**

1. Before switching to this branch, set up redirection option using previously available setting. Check if users are being redirected properly.
2. Then switch to this branch and verify if the users are being redirected as previously without any changes in settings.
3. Verify if the new settings are working properly.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Redirection after registration
